### ISSUE TITLE
is possible to access to calendar object outside app.js?

### ISSUE THREAD

**Issue by jorgeakanieves** [Serhioromano/bootstrap-calendar#518]
I´m trying to access to the calendar object from outside app.js but outside this file, window.calendar object is only html

**Comment by eleazan** [comment]
It's a scope problem... if you set the var calendar accesible from out scope of app.js... you should can use it

**Comment by jorgeakanieves** [comment]
yes, I see.... I don´t want to change the behaviour of calendar call from app.js however I have noticed that I can make some things inside Calendar.prototype functions or _update_view function...
